README.md

```diff
@@ -4,47 +4,59 @@ Read and display NIDM-Results packs in the SPM environment.
 
 ##### Usage
 
-To view NIDM results using the NIDM results viewer run the following:
- ```
- nidm_results_display(<full path to nidm json file you wish to view>)
- ```
-(where `<full path to nidm json file you wish to view>` is replaced with the full path to the nidm json file you wish to view)
+To view NIDM results using the NIDM results viewer run the following
+
+1. Open `SPM fMRI` from the Matlab command line.
+1. Click on Batch.
+
+<img src="Doc/examplemenu1.png" width="200">  |
+:-------------------------:
+ **Fig. 1.** The SPM menu. |
+3. Click on SPM -> Tools -> NIDM-Results Display
 
-This should create output like the below.
+<img src="Doc/examplemenu2.png" width="200">  |
+:-------------------------:
+ **Fig. 2.** The SPM option in the batch window. |
+4. Enter the NIDM-Results file you wish to view and where you wish the output html to be stored.
+
+<img src="Doc/example_batch.png" width="200"> |  
+:-------------------------:|
+ **Fig. 3.** The NIDM-Results batch window. |
+5. If there are multiple excursion sets stored in the NIDM-Results file the contrast window will open. Select the contrasts you wish to view and click done.
```

Comment:
Could we have the option to specify the contrast from the original batch rather than interactively opening the contrast manager? This would be similar to the behavior of 'SPM > Stats > Results Report' and would allow to run the NIDM-Results viewer completely in batch mode.